### DISCUSSION TITLE
Where and how to find and contribute projects

### DISCUSSION THREAD

**Discussion by HAQ-NAWAZ-MALIK** [community/community#112190] [Programming Help]
How exactly we contribute to open source projects , where to find , what and how will we know what we have to solve or do with it, is there any benefits, while contributing open source projects. 

What are the best Top projects in 2024 for An Beginners AI specialist student.

Does any project give any certification or badge or anything?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @HAQ-NAWAZ-MALIK ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
